_includes/includes/kb_doc_template.php

```diff
@@ -158,27 +164,48 @@ function write_version_history($pagename){
       }
       $i++;
     }
-    $string.= '</ul><br>';
+    $string.= '</ul>';
+    return $string;
+  }
+
+  /* 
+    This functions works to show the keyboard download link
+    on the keyboard help documentation of help.keyman.com
+  */
+  function download_keyboard($pagename) {
+    global $keyboardname, $kbdid; // from head() function
+    
+    if(empty($kbdid)) return '';
+    
+    if(empty($pagename)) $pagename = '';
+    
+    $string = '<h2>Download this keyboard</h2><ul id="download-keyboard">';
+            
+    $string.= "<li><a href='{$KeymanHosts->keyman_com}/keyboards/$kbdid'>Download $keyboardname</a></li>";
```

Comment:
The `KeymanHosts` reference here was incorrect, so it was emitting a blank string, making this a relative path instead of pointing to keyman.com.

```suggestion
    $string.= "<li><a href='".KeymanHosts::Instance()->keyman_com."/keyboards/$kbdid'>Download $keyboardname</a></li>";
```